### ISSUE TITLE
ohai virtualization does not detect docker on rhel 7.1

### ISSUE THREAD

**Issue by warroyo** [chef/ohai#592]
I am running test kitchen on rhel 7.1 and ubuntu trusty. I am seeing an issue where on rhel the virtualization platform is not being correctly detected. Specifically I am attempting to detect docker. here are the outputs from ohai on both Ubuntu and rhel 7.1. This is breaking cookbooks that try to detect if  it is running on docker.

the versions i am using are below:
RHEL: 
 - docker: 1.7.1 
 -Ohai: 8.3.0
 -test kitchen: 1.4.0 

ubuntu: 
  - docker: 1.7.1 
 -Ohai: 8.3.0
 -test kitchen: 1.4.0 

results from ohai

RHEL:
```
"virtualization": {
    "systems": {

    }
```

UBUNTU:
```
 "virtualization": {
    "systems": {
      "vbox": "guest",
      "docker": "guest"
    },
    "system": "docker",
    "role": "guest"
  } 
```

This seems to be an issue with the way RHEL 7.1 sets up the `/proc/self/cgroup `. 

RHEL: `8:blkio:/system.slice/docker`
UBUNTU: `10:hugetlb:/docker`

**Comment by spdqbr** [comment]
Looks like the new `/proc/self/cgroup` format breaks the regex check in `lib/ohai/plugins/linux/virtualization.rb`.

The regex `^\d+:[^:]+:/(lxc|docker)/.+$` doesn't expect anything before or after the "docker|lxc" declaration.
Quick fix is to edit the linux/virtualization.rb  and change the regex check to something like `^\d+:[^:]+:/[^:]*(lxc|docker)[-/].+$`, though I'm not certain if any other cgroup issues need to be taken into account.

**Comment by mcquin** [comment]
There's an active [pull request](https://github.com/chef/ohai/pull/569/files) which handles the new cgroup format. It needs another +1 from a chef maintainer before it can be merged.

**Comment by btm** [comment]
It looks like #569 was merged and released in Ohai v8.6.0. This also happens to be a duplicate of #531.
